pom.xml

```diff
@@ -212,27 +196,25 @@
     </pluginRepositories>
 
     <scm>
-		<connection>scm:git:ssh://github.com/jenkinsci/${project.artifactId}.git</connection>
+		<connection>scm:git:https://github.com/jenkinsci/${project.artifactId}.git</connection>
 		<developerConnection>scm:git:ssh://git@github.com/jenkinsci/${project.artifactId}.git</developerConnection>
 		<url>https://github.com/jenkinsci/${project.artifactId}</url>
  	  <tag>HEAD</tag>
   </scm>
 
     <properties>
+        <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
+        <jenkins.baseline>2.479</jenkins.baseline>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.375.4</jenkins.version>
-        <!-- Java Level to use. Java 7 required when using core >= 1.612 -->
-        <java.level>7</java.level>
+        <jenkins.version>${jenkins.baseline}.3</jenkins.version>
         <!-- Jenkins Test Harness version you use to test the plugin. -->
         <!-- For Jenkins version >= 1.580.1 use JTH 2.x or higher. -->
-        <jenkins-test-harness.version>2007.v8f9e5c18335a_</jenkins-test-harness.version>
+        <jenkins-test-harness.version>2414.v185474555e66</jenkins-test-harness.version>
```

Comment:
Usually the Jenkins test harness version is provided by the parent pom.  If there is a specific need to pin a version of the test harness, it would be good to note that with a TODO comment so that the specification of a precise test harness version can be removed when it is no longer required.
```suggestion
```